aca-docker/bin/set-environment-variables.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+## set the environment variables for MCA
+
+function set_env_variables() {
+    set_env_variables_from_file "./env_variables_all.txt"
+}
+
+function set_env_variables_from_file() {
+    file=$1
+    if [ -f ${file} ]
+    then
+        osName="$(uname -s)"
+        echo "Setting env variables from [$file] on [$osName]."
+        while IFS="=" read -r key value
+        do
+            if [[ "Darwin" == "$osName" ]];then
+                command="export $key=$value"
+                $command
```

Comment:
If this fails on EVs that have spaces in values : would switching to `source ./aca-docker/bin/env_variables_all.txt ` work either in this script or as an alternative to it?  (if so update script and/or README)